editor/src/clj/editor/workspace.clj

```diff
@@ -291,18 +292,31 @@ ordinary paths."
 (defn- absolute-path [^String path]
   (.startsWith path "/"))
 
+(defn- get-parent-dir [^String path]
+  (str (.getParent (File. path))))
+
+(defn- relative-path [^String base ^String path]
+  (let [path-parts (str/split path #"\.\.\/")
+        parent-path (if (= "/" base)
+                      base
+                      (str base "/"))]
+    (prn parent-path)
+    (if (> (count path-parts) 1)
+      (recur (get-parent-dir parent-path) (string/join "/" (drop 1 path-parts)))
+      (str parent-path path))))
+
```

Comment:
Is there some other way of doing this? Previously it just concatenated the relative path with the dir path which didn't seem to work in the editor for "../" paths

Reply:
Yes, I think we can do better. See below!